protostuff-runtime-registry/src/main/java/io/protostuff/runtime/ExplicitIdStrategy.java

```diff
@@ -44,6 +44,8 @@
 import io.protostuff.Output;
 import io.protostuff.Pipe;
 import io.protostuff.Schema;
+import com.google.common.collect.Multimap;
+import io.protostuff.MultiMapSchema;
```

Comment:
Hmm this is a problem. The registry module will have to depend on the `guava` module.
Not sure how we'll obtain adding MultiMap that doesn't have a hard dependency on guava.

Reply:
Indeed,  the problem that register depends on guava cannot be resolved for the time being.
The code files related to the `Multimap` are moved to the `protostuff-runtime-guava` module.
Please check, thank you @dyu .